readme.md

```diff
@@ -21,8 +21,15 @@ The below sample of code is in Java. Syntax may vary in other languages like kot
 
 # Step 1.1. once you have the instance of BALBTDongleLib then you need to initialize the dongle by calling the below function soon after the Step 1.
 	boolean BTDongleComm = balBTDongleLib.initBTDongleComm(bt_dongle_name);
+    # this parameter is : String bt_dongle_name
+    # Note:
+        1) invalid inputs for this function will be : null, " ", numeric input, special character
+        2) valid input example: BAL_BTDongleR2_052  
 # Step 1.2. To set the package directory so lib can access all the downloaded files in the application, call the below function call once Step 1.1 has returned true i.e. `BTDongleComm`
-  	balBTDongleLib.setPackageDir(Context context);

```

Comment:
please provide more details like you have given in above in Step 1.1
Also please please provide code reference on how they can get the `packageDir` as a string please refer to the code